[BUG] Workflow webhooks errors on empty body

### Description

Creating a webhook with an empty body just gives an error in the logs:

```
[2025-03-01 22:00:26,055] [ERROR] [paperless.handlers] Error occurred sending webhook: 'NoneType' object has no attribute 'format'
Traceback (most recent call last):
  File "/usr/src/paperless/src/documents/signals/handlers.py", line 1069, in webhook_action
    data = parse_w_workflow_placeholders(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/paperless/src/documents/templating/workflows.py", line 54, in parse_w_workflow_placeholders
    return text.format(**formatting).strip()
           ^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'format'
```

Workaround is to just put something in the body.

### Steps to reproduce

1. Create a workflow that triggers a webhook
2.  Enter a webhook URL (doesn't need to point any where, the error happens before the request is sent)
3.  Make sure not to enter anything into the _Webhook body_
4.  Trigger the workflow

### Webserver logs

```bash
[2025-03-01 22:00:26,055] [ERROR] [paperless.handlers] Error occurred sending webhook: 'NoneType' object has no attribute 'format'
Traceback (most recent call last):
  File "/usr/src/paperless/src/documents/signals/handlers.py", line 1069, in webhook_action
    data = parse_w_workflow_placeholders(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/paperless/src/documents/templating/workflows.py", line 54, in parse_w_workflow_placeholders
    return text.format(**formatting).strip()
           ^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'format'
```

### Browser logs

```bash

```

### Paperless-ngx version

2.14.7

### Host OS

Raspbian

### Installation method

Docker - official image

### System status

```json

```

### Browser

_No response_

### Configuration changes

_No response_

### Please confirm the following

- [x] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [x] This issue is not about the OCR or archive creation of a specific file(s). Otherwise, please see above regarding OCR tools.
- [x] I have already searched for relevant existing issues and discussions before opening this report.
- [x] I have updated the title field above with a concise description.

--- a/src/documents/signals/handlers.py
+++ b/src/documents/signals/handlers.py
@@ -1002,29 +1002,37 @@ def run_workflows(
             added = timezone.localtime(timezone.now())
             created = timezone.localtime(overrides.created)
 
-        subject = parse_w_workflow_placeholders(
-            action.email.subject,
-            correspondent,
-            document_type,
-            owner_username,
-            added,
-            filename,
-            current_filename,
-            created,
-            title,
-            doc_url,
+        subject = (
+            parse_w_workflow_placeholders(
+                action.email.subject,
+                correspondent,
+                document_type,
+                owner_username,
+                added,
+                filename,
+                current_filename,
+                created,
+                title,
+                doc_url,
+            )
+            if action.email.subject
+            else ""
         )
-        body = parse_w_workflow_placeholders(
-            action.email.body,
-            correspondent,
-            document_type,
-            owner_username,
-            added,
-            filename,
-            current_filename,
-            created,
-            title,
-            doc_url,
+        body = (
+            parse_w_workflow_placeholders(
+                action.email.body,
+                correspondent,
+                document_type,
+                owner_username,
+                added,
+                filename,
+                current_filename,
+                created,
+                title,
+                doc_url,
+            )
+            if action.email.body
+            else ""
         )
         try:
             n_messages = send_email(
@@ -1105,7 +1113,7 @@ def run_workflows(
                             f"Error occurred parsing webhook params: {e}",
                             extra={"group": logging_group},
                         )
-            else:
+            elif action.webhook.body:
                 data = parse_w_workflow_placeholders(
                     action.webhook.body,
                     correspondent,

